res/openvpn-gui-res-it.rc

```diff
@@ -332,127 +347,126 @@ Completa la richiesta di autorizzazione precedente."
 Rimarrai connesso anche se esci dall'interfaccia di OpenVPN.\n\n\
-Vuoi continuare e uscire da dall'interfaccia di OpenVPN?"
+Vuoi continuare ed uscire da dall'interfaccia di OpenVPN?"
 
     /* options - Resources */
-    IDS_NFO_USAGE "--help\t\t\t: Mostra questo messaggio.\n\
---connect cnn \t\t: Connetti a ""cnn"" all'avvio. (va inclusa l'estensione)\n\
+    IDS_NFO_USAGE "--help\t\t\t: Visualizza questo messaggio.\n\
+--connect cnn \t\t: All'avvio connettiti a ""cnn"" (va inclusa l'estensione).\n\
 \t\t\t   Esempio: openvpn-gui --connect office.ovpn\n\
 --command cmd [args]\t: Invia un comando a un'istanza aperta dell'interfaccia.\n\
 Comandi supportati:\n\
-    connect cnn            \t: connetti la configurazione ""cnn""\n\
-    disconnect cnn         \t: disconnetti la configurazione ""cnn""\n\
-    reconnect cnn          \t: riconnetti la configurazione ""cnn""\n\
-    disconnect_all         \t: disconnetti tutte le configurazioni connesse\n\
-    exit                 \t\t: termina l'istanza aperta dell'interfaccia (può chiedere conferma)\n\
-    status cnn           \t\t: mostra lo stato della configurazione ""cnn"" se connessa\n\
+    connect cnn            \t: connetti usando la configurazione ""cnn""\n\
+    disconnect cnn         \t: disconnetti usando la configurazione ""cnn""\n\
+    reconnect cnn          \t: riconnetti usando la configurazione ""cnn""\n\
+    disconnect_all         \t: disconnetti tutte le connessioni aperte\n\
+    exit                 \t\t: termina l'istanza aperta dell'interfaccia (può richiedere conferma)\n\
+    status cnn           \t\t: visualizza lo stato della configurazione ""cnn"" se connessa\n\
     silent_connection [0|1]\t: imposta la flag silent_connection on (1) oppure off (0)\n\
-    import path          \t\t: Import the config file pointed to by path\n\
+    import path          \t\t: Importa il file di configurazione puntato dal percorso\n\
 \t\t\tEsempio: openvpn-gui.exe --command disconnect myconfig\n\
 \n\
-Opzioni per ignorare il registro di sistema:\n\
---exe_path\t\t: Percorso di openvpn.exe.\n\
---config_dir\t\t: Percorso della cartella dei file di configurazione.\n\
---ext_string\t\t: Estensione dei file di configurazione.\n\
---log_dir\t\t\t: Percorso della cartella dei file di log.\n\
---priority_string\t\t: Stringa di priorità (Vedi install.txt per maggiori informazioni).\n\
---append_string\t\t: 1=Aggiungi in coda al log. 0=Tronca log alla connessione.\n\
---log_viewer\t\t: Percorso del visualizzatore log.\n\
---editor\t\t\t: Percorso dell'editor di configurazione.\n\
---allow_edit\t\t: 1=Mostra opzione modifica configurazione.\n\
---allow_service\t\t: 1=Mostra menu controllo servizio.\n\
---allow_password\t\t: 1=Mostra menu modifica password.\n\
---allow_proxy\t\t: 1=Mostra menu impostazioni proxy.\n\
---show_balloon\t\t: 0=Mai, 1=Alla connessione iniziale, 2=A ogni riconnessione.\n\
+Opzioni per sovrascrivere le impostazioni:\n\
```

Comment:
I don't think "sovrascrivere" is a correct translation for "override", because this is just a "in place replacement" of the original value for the options, but it's hard to find a one word translation....not sure what to suggest

Reply:
ovveride it'f foce e replacement of a setting.
Then it's correct for me "sovarscrivere" the sertting set in otehr place.